upgrade/UPGRADE-v9.2.0-dev.md

```diff
@@ -8,6 +8,22 @@ There you can find links to upgrade notes for other versions too.
 ## Composer dependencies
 - remove `vasek-purchart/console-errors-bundle` dependency ([#2408](https://github.com/shopsys/shopsys/pull/2408))
     - see #project-base-diff to update your project
+- upgrade your dependencies to the versions that support both PHP 7.4 and 8.1 ([#2417](https://github.com/shopsys/shopsys/pull/2417))
+    - see #project-base-diff to update your project
+    - make sure you check all the upgrading notes of the upgraded packages and adjust your codebase appropriately
+    - `DatabaseDumpCommand::__construct` changed its interface:
+        ```
+        - public function __construct(Connection $connection)
+        + public function __construct(DatabaseConnectionCredentialsProvider $databaseConnectionCredentialsProvider)
+        ```
+    - `EntityExtensionSubscriber::getClassMetadataForEntity` changed its interface:
+        ```diff
+        - protected function getClassMetadataForEntity(string $entityClass): ClassMetadataInfo
+        + protected function getClassMetadataForEntity(string $entityClass): ClassMetadata
+        ```
+    - `Admin/AdvertController` has a new dependency on `EntityManagerInterface` in the constructor
+    - `Admin/BrandController` has a new dependency on `EntityManagerInterface` in the constructor
+    - `Admin/SliderController` has a new dependency on `EntityManagerInterface` in the constructor
```

Comment:
Is removal of the parameter from the flush method worth mentioning?

Reply:
I do not think so - in such a case, I would feel like I need to mention all the other changes as well, I believe the combination of the project-base diff along with all the packages upgrade notes is sufficient